inc/common/purge.php

```diff
@@ -83,10 +83,10 @@ function rocket_get_purge_urls( $post_id, $post ) {
 			if ( is_ssl() ) {
 				$filename .= '-https';
 			}
-
+			global $is_nginx;
```

Comment:
Not used later? (same observation for other places)

Reply:
Thank you! I wanted to removed those, but I forgot on the way